src/data/classskills.txt

```diff
@@ -34,7 +34,7 @@
 # Skill names have changed, keeping this for historical reasons only
 
 # 16	Summon Snowcone		1	5	0
-# 17	Summon Hilarious Objects		1	5	0
+17	Summon Hilarious Objects		1	5	0
```

Comment:
Absolutely not.

```
7226	Summon Hilarious Objects	book2.gif	1	5	0
```

What do you get if you look up "Summon Hilarious Objects" by name.

Keep this as a comment.

Reply:
Multiple matches.

Reply:
Reverted.  Will direct my sense of something needs to be fixed by adding to the CharSheet testing.